minimalTest.js

```diff
@@ -53,7 +52,10 @@ function setRoomList() {
         }
         return 0;
     });
-    roomList = roomList.reverse()
+    tmp = tmp.reverse()
```

Comment:
if you just flip the signs of all the returns in the sorting function above, you won't need to `reverse()` here